### PR TITLE
docs - api docs for 44

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
- Updates API docs.
- Updates code that generates API docs to handle formatting of source code descriptions for YAML frontmatter.

### CODE REVIEW

File: docs/api/activity.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: "Activity"
-summary: "API endpoints for Activity."
+summary: |
```
Comment (nllho) [inline docs/api/activity.md:3]: just for my own knowledge -- what does the upright slash do?

File: docs/api/embed.md [hunk 2]
```diff
@@ -1,20 +1,7 @@
...
-    2.  Sign/base-64 encode a JSON Web Token using the secret key and pass it as the relevant part of the URL path
-        to the various endpoints here.
-
-   Tokens can have the following fields:
-
-      {:resource {:question  <card-id>
-                  :dashboard <dashboard-id>}
-       :params   <params>}."
+summary: |
+  Various endpoints that use [JSON web tokens](https://jwt.io/introduction/) to fetch Cards and Dashboards. The endpoints are the same as the ones in `api/public/`, and differ only in the way they are authorized.  To use these endpoints:  1.  Set the `embedding-secret-key` Setting to a hexadecimal-encoded 32-byte sequence (i.e., a 64-character string). You can use `/api/util/random_token` to get a cryptographically-secure value for this. 2.  Sign/base-64 encode a JSON Web Token using the secret key and pass it as the relevant part of the URL path to the various endpoints here.  Tokens can have the following fields:  {:resource {:question  <card-id> :dashboard <dashboard-id>} :params   <params>}.
```
Comment (nllho) [inline docs/api/embed.md:4]: Looks like the numbering format has been lost here?

**Review by nllho — APPROVED** [review]

File: docs/api/embed.md (hunk 2 above)
Comment (jeff-bruemmer) [reply docs/api/embed.md:4]: That's intentional, since it's for frontmatter descriptions.

File: docs/api/activity.md (hunk 1 above)
Comment (jeff-bruemmer) [reply docs/api/activity.md:3]: https://yaml.org/spec/1.2.2/#:~:text=Shoes%0A%20%20quantity%3A%201-,2.3.%20Scalars,-Scalar%20content%20can

Probably not necessary now that I'm losing the line breaks to format descriptions as a single line.

**Pull request merged**